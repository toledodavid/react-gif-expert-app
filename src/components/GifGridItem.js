import React from 'react';


const GifGridItem = ({title, imgUrl}) => {

  return(
    <div className="card animate__animated animate__fadeIn">
      <img src={imgUrl} alt={title} />
      <p>{title}</p>
    </div>
  );
}


export default GifGridItem;