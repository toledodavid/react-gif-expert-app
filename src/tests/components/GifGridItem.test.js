import '@testing-library/jest-dom';
import React from 'react';
import { shallow } from 'enzyme';
import GifGridItem from '../../components/GifGridItem';

describe('Tests for <GifGridItem/> component', () => {
  let wrapper = shallow(<GifGridItem />);

  beforeEach(() => {
    wrapper = shallow( <GifGridItem />);
  });

  test('It should render <GifGridItem/> correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});