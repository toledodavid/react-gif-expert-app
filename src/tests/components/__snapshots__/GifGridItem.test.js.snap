// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for <GifGridItem/> component It should render <GifGridItem/> correctly 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img />
  <p />
</div>
`;
